packages/algob/test/builtin-tasks/run.ts

```diff
@@ -193,19 +193,19 @@ scripts directory: script 1 executed
     assert.equal(scriptOutput, "deployASA script\n");
   });
 
-  it("Should crash on fundLsig", async function () {
+  it("Should crash on fundLsigByFile", async function () {
```

Comment:
How about the case `crash when use fundLsig, deployApp...` on for `run mode`?

Reply:
we can't use byname for this test as `mkContractLsig` runs in deploy mode. This test executes in RUN mode :)